### PR TITLE
feat: add nessie

### LINKED ISSUE #105: Add Nessie server
Add playbooks to deploy [Nessie](https://projectnessie.org/) server.

### PR DESCRIPTION
**Pull request by sergkudinov**
Depends on:
- https://github.com/TOSIT-IO/tdp-collection-extras/pull/25

Please, merge it first.

#### Which issue(s) this PR fixes:

Fixes #105

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Launch a fully-featured virtual TDP Hadoop cluster with a single command _or_ cu
 - The Python package `jmespath` (an Ansible dependency for JSON parsing/querying)
 - Unzip (to execute the setup scripts)
 - `jq` required to execute helper script
+- `realpath` utility
```
Comment (DanielJohnHarty) [inline README.md:21]: Is this used on the Ansible control node? If it is a requirement of nessie it should be installed in the tdp-collection-extras nessie role.

You can also remove the dependency and use a bash command like in lines 9 and 10 of [the setup.sh](https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/setup.sh)

Comment (sergkudinov) [reply README.md:21]: It is actually a requirement for the setup.sh script - https://github.com/TOSIT-IO/tdp-getting-started/blob/master/scripts/setup.sh#L10

On my mac it didn't work until I installed `realpath`.

Tell me if I need to exclude this line from this PR and create another one.

**Comment by mehdibn** [comment]
@sergkudinov thank you for your contribution to TDP!
In a fully open-source approach, all Issues/Pull Requests are fairly examined for integration.
Your PR integrates a new technology to TDP. Can you please open a **new discussion** for validation before any implementation. See the [TDP new component proposal guidelines](https://github.com/orgs/TOSIT-IO/discussions/25) for details on how to organize your request.
Thank you again for your contribution to TDP.

**Comment by leopaul36** [comment]
Closing as stale

**Pull request closed without merging**